LEAF_Request_Portal/sources/FormWorkflow.php

```diff
@@ -519,8 +519,6 @@ public function handleAction($dependencyID, $actionType, $comment)
                     break;
                 default:
                     return array('status' => 0, 'errors' => array('Invalid Operation'));
-
-                    break;
```

Comment:
Do we want to leave breaks on defaults or can we remove them?

Reply:
This is removed because the code can never get to it. I stops with the return. If the return isn't there I would say you'd want he break.